script.js

```diff
@@ -1,12 +1,77 @@
 //You can edit ALL of the code here
+
 function setup() {
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
-
+//Level 100
 function makePageForEpisodes(episodeList) {
   const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  rootElem.innerHTML = "";
```

Comment:
Question: What's the purpose of setting innerHTML to blank here?

Reply:
When I call this function on the search input, if I don't "reset" the root div, it will show the searched episode at the end of the page instead of those episodes being the only ones changed.